server/plugin/webhook.go

```diff
@@ -184,6 +184,7 @@ func (wb *WebhookBroker) Close() {
 }
 
 func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
+	p.API.LogDebug("Received webhook event.")
```

Comment:
To avoid over spamming of the logs, I think we should make sure the repo is https://github.com/mattermost/docs before logging anything. We can have a function `p.logForDocsRepoDebugging(repo, message)` to but this logic there.

I think line 216 is the first point where we have access to which repo the event is for